Include/internal/pycore_pyatomic_ft_wrappers.h

```diff
@@ -156,6 +158,7 @@ extern "C" {
 #define FT_ATOMIC_STORE_LLONG_RELAXED(value, new_value) value = new_value
 #define FT_ATOMIC_LOAD_ULLONG_RELAXED(value) value
 #define FT_ATOMIC_STORE_ULLONG_RELAXED(value, new_value) value = new_value
+#define FT_ATOMIC_ADD_SSIZE(value, new_value) (value += new_value) - new_value
```

Comment:
There are many warnings about this macro. I don't think that it's a good idea to compute the old value like that. I suggest something like:

```c
static inline Py_ssize_t
FT_ATOMIC_ADD_SSIZE(Py_ssize_t *obj, Py_ssize_t value)
{
    Py_ssize_t result = *obj;
    *obj += value;
    return result;
}
```

As I wrote previously, I would prefer to have no return value instead.

Reply:
Is what I am doing at the moment.

Reply:
I agree about no return value, but I don't think we should use an inline function here -- the macros don't take pointers: `value` is an l-value.

Reply:
Look at latest, it fits usage pattern.

Reply:
I would also be fine with @colesbury's macro: `#define FT_ATOMIC_ADD_SSIZE(value, new_value) (void)(value += new_value)`.

Reply:
Otherwise just tell me what u want there and I'll put.

Reply:
No return value it is then.